changelogs/unreleased/gh-10363-datetime-handle-timezone.md

```diff
@@ -0,0 +1,5 @@
+## bugfix/datetime
```

Comment:
The comment should be closed after adding a doc request.

Reply:
Doc request tarantool/doc#4720

Reply:
Usually we write doc request in commit message body (like this https://github.com/tarantool/tarantool/commit/4c302320c40f7ec2c81b10bf72287bb5779ab43b). Issue in doc/doc repository will be created automatically after merging.  However, a manually created issue is ok too.